packages/docusaurus-plugin-content-docs/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "lib/index.js",
   "sideEffects": false,
   "exports": {
+    "./src/*": "./src/*",
```

Comment:
This is needed, because we map `/client` to `/src/client` instead of `lib`, so we don't have to re-build. It shouldn't be problematic anyways.